config/m4/compiler.m4

```diff
@@ -339,6 +339,30 @@ AC_LANG_POP
 AM_CONDITIONAL([HAVE_CXX11], [test "x$cxx11_happy" != xno])
 
 
+#
+# PGI specific switches
+#
+ADD_COMPILER_FLAG_IF_SUPPORTED([--display_error_number],
+                               [--display_error_number],
+                               [AC_LANG_SOURCE([[int main(){return 0;}]])],
+                               [],
+                               [])
+
+# Suppress incorrect printf format. TODO: remove it after compiler fix
```

Comment:
Suppress incorrect printf format for PGI18 compiler

Reply:
comment updated